applications/visualizer/frontend/src/components/viewers/EM/EMStackTilesViewer.tsx

```diff
@@ -84,11 +41,62 @@ const newSegLayer = (dataset: Dataset, slice: number) => {
       url: getSegmentationURL(dataset, slice),
       format: new GeoJSON(),
     }),
-    style: getFeatureStyle,
     zIndex: 1,
   });
 };
 
+function neuronsStyle(feature: FeatureLike, workspace: Workspace) {
+  const neuronName = neuronFeatureName(feature);
+
+  const visibleNeurons = workspace.getVisibleNeuronsInEM();
+  const visibleNeuron = visibleNeurons.find((neuron) => neuron === neuronName);
+  const color = visibleNeuron ? workspace.visibilities[neuronName][ViewerType.EM].color : undefined;
+
+  const selectedNeuron = workspace.getViewerSelectedNeurons(ViewerType.EM).find((neuron) => neuron === neuronName);
+  if (selectedNeuron) {
+    return selectedNeuronStyle(feature, color);
+  }
+
+  if (visibleNeuron) {
+    return activeNeuronStyle(feature, color);
+  }
+
+  return null;
+}
+
+function isNeuronVisible(neuronId: string, workspace: Workspace): boolean {
```

Comment:
I prefered the version before when this function was defined with the two functions `isNeuronActive` and `isNeuronVisible` :D, it was easier to read at the end as you would have:

```typescript
function isNeuronVisible(...) {
	return isNeuronActive(...) || isNeuronSelected(...)
}
```
Also, better use `includes` instead of `find` in those cases also.